fix(Embeddings Azure OpenAi Node): Add proxy agent

vicalca opened the pull request:
## Summary

Solved issue where the EmbeddingsAzureOpenAi node times out behind a proxy configuration.

Added `proxyAgent` to the Embeddings Azure OpenAi Node, like it was done in #15449 to solve a similar issue with the chat models.

## Related Linear tickets, Github issues, and Community forum posts

linear ticket: [GHC-3339](https://linear.app/n8n/issue/GHC-3339)

resolves #17688

## Review / Merge checklist

- [x] PR title and summary are descriptive.
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [x] Tests included.
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

<!-- CURSOR_SUMMARY -->
---

> [!NOTE]
> Wire proxy agent into Azure OpenAI Embeddings requests and add a test to validate dispatcher usage.
> 
> - **Embeddings**:
>   - Configure `EmbeddingsAzureOpenAi` to pass `configuration.fetchOptions.dispatcher` from `getProxyAgent` to `AzureOpenAIEmbeddings`, using the endpoint or `resourceName`.
> - **Tests**:
>   - Add unit test `EmbeddingsAzureOpenAi.test.ts` verifying the embeddings client is constructed with a proxy `dispatcher`.
> 
> <sup>Written by [Cursor Bugbot](https://cursor.com/dashboard?tab=bugbot) for commit cbc1565a8f57332b805cd0b2080f8752ddc0cc29. This will update automatically on new commits. Configure [here](https://cursor.com/dashboard?tab=bugbot).</sup>
<!-- /CURSOR_SUMMARY -->

vicalca committed:
fix(Embeddings Azure OpenAi Node): add proxy agent

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/n8n-io/n8n?pullRequest=18663) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/n8n-io/n8n?pullRequest=18663) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/n8n-io/n8n?pullRequest=18663) it.</sub>

Joffcom commented:
Hey @vicalca,

Thank you for your contribution. We appreciate the time and effort you’ve taken to submit this pull request.

Before we can proceed, please ensure the following:
	•	Tests are included for any new functionality, logic changes or bug fixes.
	•	The PR aligns with our contribution guidelines.

Regarding new nodes:
We no longer accept new nodes directly into the core codebase. Instead, we encourage contributors to follow our Community Node Submission Guide to publish nodes independently.

If your node integrates with an AI service that you own or represent, please email nodes@n8n.io and we will be happy to discuss the best approach.

About review timelines:
This PR has been added to our internal tracker as "GHC-3944". While we plan to review it, we are currently unable to provide an exact timeframe. Our goal is to begin reviews within a month, but this may change depending on team priorities. We will reach out when the review begins.

Thank you again for contributing to n8n.

vicalca committed:
fix(Embeddings Azure OpenAi Node): add targetUrl

vicalca commented:
@cubic-dev-ai Implemented feedback.

danny22-seo commented:
Hi, is there any update or timeline for fixing this issue? I'm waiting on a quick resolution, would be greatly appreciated.

hgabor47 commented:
Azure is primarily used by businesses, so this is quite a major inconvenience and could lead to the loss of corporate customers.   
Azure OpenAI Embedding is not working for me either.  
What is the priority for releasing a fix for this bug?

vicalca committed:
test(EmbeddingsAzureOpenAi Node): add test proxy

Add test to check if proxyAgent is being used as dispatcher for the
underlying AzureOpenAiEmbeddings.

vicalca commented:
@Joffcom @danny22-seo Added unit test.

danny22-seo commented:
@RomanDavydchuk 
Hi, Is there any update?, Would you check this problem and fix?

danny22-seo commented:
@RomanDavydchuk 
I've test on n8n 1.117.3, but still it's not working. 
I use n8n on docker container, setup like below 
services:
  n8n:
    image: docker.n8n.io/n8nio/n8n:1.117.3
    environment:
      - N8N_HOST=aicamp.company.com
      - N8N_LISTEN_ADDRESS=0.0.0.0
      - N8N_PORT=8080
      - N8N_ENFORCE_SETTINGS_FILE_PERMISSIONS=true
      - N8N_SECURE_COOKIE=false
      - N8N_DIAGNOSTICS_ENABLED=false
      - N8N_VERSION_NOTIFICATIONS_ENABLED=false
      - N8N_TEMPLATES_ENABLED=false
      - N8N_DIAGNOSTICS_CONFIG_FRONTEND=
      - N8N_DIAGNOSTICS_CONFIG_BACKEND=
      - N8N_LOG_LEVEL=debug
      - EXTERNAL_FRONTEND_HOOKS_URLS=
      - HTTP_PROXY=http://proxydev.company.com:8080
      - HTTPS_PROXY=http://proxydev.company.com:8080   

I use both Azure gpt 4.1 api and Azure text embedding api on my workflow. 
Azure gpt 4.1 is working well, but text embedding api is still not working. 
I've got Error message "Connection error" on web ui.

this is debug level container log.
n8n-1  | 2025-11-03T07:35:23.837Z | debug | Initializing event bus... {"file":"message-event-bus.js","function":"initialize"}
n8n-1  | 2025-11-03T07:35:23.845Z | debug | Initializing event writer {"file":"message-event-bus.js","function":"initialize"}
n8n-1  | 2025-11-03T07:35:23.849Z | debug | Checking for unsent event messages {"file":"message-event-bus.js","function":"initialize"}
n8n-1  | 2025-11-03T07:35:23.903Z | debug | Start logging into /home/node/.n8n/n8nEventLog.log  {"file":"message-event-bus.js","function":"initialize"}
n8n-1  | 2025-11-03T07:35:23.905Z | debug | MessageEventBus initialized {"file":"message-event-bus.js","function":"initialize"}
n8n-1  | 2025-11-03T07:35:23.909Z | debug | Installing global HTTP proxy agents {"HTTP_PROXY":"http://proxydev.company.com:8080","HTTPS_PROXY":"http://proxydev.company.com:8080","file":"logger-proxy.js","function":"exports.debug"}
n8n-1  | 2025-11-03T07:35:23.910Z | info | Version: 1.117.3 {"file":"abstract-server.js","function":"start"}
n8n-1  | 2025-11-03T07:35:23.910Z | debug | Server ID: main-d4a9ef5eb00b {"file":"server.js","function":"start"}
n8n-1  | 2025-11-03T07:35:23.911Z | debug | Soft-deletion every 60 minutes {"scopes":["pruning"],"file":"executions-pruning.service.js","function":"scheduleRollingSoftDeletions"}
n8n-1  | 2025-11-03T07:35:23.911Z | debug | Hard-deletion in next 15 minutes {"scopes":["pruning"],"file":"executions-pruning.service.js","function":"scheduleNextHardDeletion"}
n8n-1  | 2025-11-03T07:35:23.912Z | debug | Started pruning timers {"scopes":["pruning"],"file":"executions-pruning.service.js","function":"startPruning"}
n8n-1  | 2025-11-03T07:35:23.923Z | info | Start Active Workflows: {"scopes":["workflow-activation"],"file":"active-workflow-manager.js","function":"addActiveWorkflows"}
n8n-1  | 2025-11-03T07:35:26.199Z | debug | Added webhooks for workflow "HR V2" (ID fksCUlSghdpu9QKb) {"scopes":["workflow-activation"],"workflowId":"fksCUlSghdpu9QKb","file":"active-workflow-manager.js","function":"addWebhooks"}
n8n-1  | 2025-11-03T07:35:26.203Z | debug | Added triggers and pollers for workflow "HR V2" (ID: fksCUlSghdpu9QKb) {"scopes":["workflow-activation"],"file":"active-workflow-manager.js","function":"addTriggersAndPollers"}
n8n-1  | 2025-11-03T07:35:26.206Z | info | Activated workflow "HR V2" (ID: fksCUlSghdpu9QKb) {"scopes":["workflow-activation"],"workflowName":"HR V2","workflowId":"fksCUlSghdpu9QKb","file":"active-workflow-manager.js","function":"activateWorkflow"}
n8n-1  | 2025-11-03T07:35:26.206Z | debug | Finished activating all workflows {"scopes":["workflow-activation"],"file":"active-workflow-manager.js","function":"addActiveWorkflows"}
n8n-1  | 2025-11-03T07:35:26.207Z | info | 
n8n-1  | Editor is now accessible via:
n8n-1  | http://aicamp.company.com:8080 {"file":"base-command.js","function":"log"}
n8n-1  | 2025-11-03T07:35:34.376Z | debug | Skipped browserId check on /types/credentials.json {"file":"auth.service.js","function":"resolveJwt"}
n8n-1  | 2025-11-03T07:35:34.399Z | debug | Skipped browserId check on /types/nodes.json {"file":"auth.service.js","function":"resolveJwt"}
n8n-1  | 2025-11-03T07:35:34.406Z | debug | Skipped browserId check on /rest/push {"file":"auth.service.js","function":"resolveJwt"}
n8n-1  | 2025-11-03T07:35:34.407Z | debug | Add editor-UI session {"pushRef":"rjpmbwqxpe","file":"abstract.push.js","function":"add"}
n8n-1  | 2025-11-03T07:35:42.904Z | debug | Execution added {"executionId":"91","file":"active-executions.js","function":"add"}
n8n-1  | 2025-11-03T07:35:42.918Z | debug | Execution for workflow HR V2 was assigned id 91 {"executionId":"91","file":"workflow-runner.js","function":"runMainProcess"}
n8n-1  | 2025-11-03T07:35:42.931Z | debug | Execution ID 91 had triggerToStartFrom. Starting from that trigger. {"executionId":"91","file":"manual-execution.service.js","function":"runManually"}
n8n-1  | 2025-11-03T07:35:42.935Z | debug | Workflow execution started {"workflowId":"fksCUlSghdpu9QKb","file":"logger-proxy.js","function":"exports.debug"}
n8n-1  | 2025-11-03T07:35:42.943Z | debug | Executing hook (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:35:42.944Z | debug | Pushed to frontend: executionStarted {"dataType":"executionStarted","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:35:42.954Z | debug | Start executing node "AI Agent" {"node":"AI Agent","workflowId":"fksCUlSghdpu9QKb","file":"logger-proxy.js","function":"exports.debug"}
n8n-1  | 2025-11-03T07:35:42.956Z | debug | Executing hook on node "AI Agent" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:35:42.957Z | debug | Pushed to frontend: nodeExecuteBefore {"dataType":"nodeExecuteBefore","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:35:42.958Z | debug | Running node "AI Agent" started {"node":"AI Agent","workflowId":"fksCUlSghdpu9QKb","file":"logger-proxy.js","function":"exports.debug"}
n8n-1  | 2025-11-03T07:35:42.961Z | debug | Executing Tools Agent V2 {"file":"execute.js","function":"toolsAgentExecute"}
n8n-1  | 2025-11-03T07:35:43.072Z | info | Using API Key authentication for Azure OpenAI. {"file":"api-key.js","function":"setupApiKeyAuthentication"}
n8n-1  | 2025-11-03T07:35:43.072Z | info | Instantiating AzureChatOpenAI model with deployment: gpt-4.1 {"file":"LmChatAzureOpenAi.node.js","function":"supplyData"}
n8n-1  | 2025-11-03T07:35:43.078Z | info | Azure OpenAI client initialized for deployment: gpt-4.1 {"file":"LmChatAzureOpenAi.node.js","function":"supplyData"}
n8n-1  | 2025-11-03T07:35:43.105Z | debug | Supply data for embeddings {"file":"EmbeddingsAzureOpenAi.node.js","function":"supplyData"}
n8n-1  | 2025-11-03T07:35:43.123Z | debug | Executing hook on node "Simple Memory" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:35:43.123Z | debug | Pushed to frontend: nodeExecuteBefore {"dataType":"nodeExecuteBefore","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:35:43.126Z | debug | Executing hook on node "Simple Memory" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:35:43.126Z | debug | Pushed to frontend: nodeExecuteAfter {"dataType":"nodeExecuteAfter","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:35:43.128Z | debug | Pushed to frontend: nodeExecuteAfterData {"dataType":"nodeExecuteAfterData","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:35:44.247Z | debug | Executing hook on node "Azure gpt-4.1" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:35:44.247Z | debug | Pushed to frontend: nodeExecuteBefore {"dataType":"nodeExecuteBefore","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:35:46.230Z | debug | Executing hook on node "Azure gpt-4.1" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:35:46.231Z | debug | Pushed to frontend: nodeExecuteAfter {"dataType":"nodeExecuteAfter","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:35:46.232Z | debug | Pushed to frontend: nodeExecuteAfterData {"dataType":"nodeExecuteAfterData","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:35:46.242Z | debug | Executing hook on node "RAG HR" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:35:46.242Z | debug | Pushed to frontend: nodeExecuteBefore {"dataType":"nodeExecuteBefore","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | Api key is used with unsecure connection.
n8n-1  | 2025-11-03T07:35:46.278Z | debug | Executing hook on node "Embeddings Azure OpenAI" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:35:46.279Z | debug | Pushed to frontend: nodeExecuteBefore {"dataType":"nodeExecuteBefore","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:36:20.739Z | debug | Querying database for waiting executions {"scopes":["waiting-executions"],"file":"wait-tracker.js","function":"getWaitingExecutions"}
n8n-1  | 2025-11-03T07:37:20.739Z | debug | Querying database for waiting executions {"scopes":["waiting-executions"],"file":"wait-tracker.js","function":"getWaitingExecutions"}
n8n-1  | 2025-11-03T07:37:36.283Z | debug | Executing hook on node "Embeddings Azure OpenAI" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:37:36.284Z | debug | Pushed to frontend: nodeExecuteAfter {"dataType":"nodeExecuteAfter","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:36.286Z | debug | Pushed to frontend: nodeExecuteAfterData {"dataType":"nodeExecuteAfterData","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:36.289Z | debug | Executing hook on node "RAG HR" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:37:36.289Z | debug | Pushed to frontend: nodeExecuteAfter {"dataType":"nodeExecuteAfter","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:36.290Z | debug | Pushed to frontend: nodeExecuteAfterData {"dataType":"nodeExecuteAfterData","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:36.323Z | debug | Executing hook on node "Azure gpt-4.1" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:37:36.323Z | debug | Pushed to frontend: nodeExecuteBefore {"dataType":"nodeExecuteBefore","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:40.443Z | debug | Executing hook on node "Azure gpt-4.1" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:37:40.444Z | debug | Pushed to frontend: nodeExecuteAfter {"dataType":"nodeExecuteAfter","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:40.444Z | debug | Pushed to frontend: nodeExecuteAfterData {"dataType":"nodeExecuteAfterData","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:40.448Z | debug | Executing hook on node "Simple Memory" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:37:40.449Z | debug | Pushed to frontend: nodeExecuteBefore {"dataType":"nodeExecuteBefore","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:40.451Z | debug | Executing hook on node "Simple Memory" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:37:40.451Z | debug | Pushed to frontend: nodeExecuteAfter {"dataType":"nodeExecuteAfter","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:40.452Z | debug | Pushed to frontend: nodeExecuteAfterData {"dataType":"nodeExecuteAfterData","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:40.458Z | debug | Running node "AI Agent" finished successfully {"node":"AI Agent","workflowId":"fksCUlSghdpu9QKb","file":"logger-proxy.js","function":"exports.debug"}
n8n-1  | 2025-11-03T07:37:40.460Z | debug | Executing hook on node "AI Agent" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:37:40.460Z | debug | Pushed to frontend: nodeExecuteAfter {"dataType":"nodeExecuteAfter","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:40.460Z | debug | Pushed to frontend: nodeExecuteAfterData {"dataType":"nodeExecuteAfterData","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:40.461Z | debug | Start executing node "Edit Fields1" {"node":"Edit Fields1","workflowId":"fksCUlSghdpu9QKb","file":"logger-proxy.js","function":"exports.debug"}
n8n-1  | 2025-11-03T07:37:40.461Z | debug | Executing hook on node "Edit Fields1" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:37:40.461Z | debug | Pushed to frontend: nodeExecuteBefore {"dataType":"nodeExecuteBefore","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:40.462Z | debug | Running node "Edit Fields1" started {"node":"Edit Fields1","workflowId":"fksCUlSghdpu9QKb","file":"logger-proxy.js","function":"exports.debug"}
n8n-1  | 2025-11-03T07:37:40.497Z | debug | Running node "Edit Fields1" finished successfully {"node":"Edit Fields1","workflowId":"fksCUlSghdpu9QKb","file":"logger-proxy.js","function":"exports.debug"}
n8n-1  | 2025-11-03T07:37:40.498Z | debug | Executing hook on node "Edit Fields1" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:37:40.498Z | debug | Pushed to frontend: nodeExecuteAfter {"dataType":"nodeExecuteAfter","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:40.498Z | debug | Pushed to frontend: nodeExecuteAfterData {"dataType":"nodeExecuteAfterData","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:40.499Z | debug | Start executing node "Append row in sheet" {"node":"Append row in sheet","workflowId":"fksCUlSghdpu9QKb","file":"logger-proxy.js","function":"exports.debug"}
n8n-1  | 2025-11-03T07:37:40.499Z | debug | Executing hook on node "Append row in sheet" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:37:40.499Z | debug | Pushed to frontend: nodeExecuteBefore {"dataType":"nodeExecuteBefore","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:40.500Z | debug | Running node "Append row in sheet" started {"node":"Append row in sheet","workflowId":"fksCUlSghdpu9QKb","file":"logger-proxy.js","function":"exports.debug"}
n8n-1  | 2025-11-03T07:37:43.942Z | debug | Running node "Append row in sheet" finished successfully {"node":"Append row in sheet","workflowId":"fksCUlSghdpu9QKb","file":"logger-proxy.js","function":"exports.debug"}
n8n-1  | 2025-11-03T07:37:43.945Z | debug | Executing hook on node "Append row in sheet" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:37:43.945Z | debug | Pushed to frontend: nodeExecuteAfter {"dataType":"nodeExecuteAfter","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:43.945Z | debug | Pushed to frontend: nodeExecuteAfterData {"dataType":"nodeExecuteAfterData","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:43.946Z | debug | Start executing node "Edit Fields" {"node":"Edit Fields","workflowId":"fksCUlSghdpu9QKb","file":"logger-proxy.js","function":"exports.debug"}
n8n-1  | 2025-11-03T07:37:43.947Z | debug | Executing hook on node "Edit Fields" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:37:43.947Z | debug | Pushed to frontend: nodeExecuteBefore {"dataType":"nodeExecuteBefore","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:43.947Z | debug | Running node "Edit Fields" started {"node":"Edit Fields","workflowId":"fksCUlSghdpu9QKb","file":"logger-proxy.js","function":"exports.debug"}
n8n-1  | 2025-11-03T07:37:43.949Z | debug | Running node "Edit Fields" finished successfully {"node":"Edit Fields","workflowId":"fksCUlSghdpu9QKb","file":"logger-proxy.js","function":"exports.debug"}
n8n-1  | 2025-11-03T07:37:43.950Z | debug | Executing hook on node "Edit Fields" (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:37:43.950Z | debug | Pushed to frontend: nodeExecuteAfter {"dataType":"nodeExecuteAfter","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:43.951Z | debug | Pushed to frontend: nodeExecuteAfterData {"dataType":"nodeExecuteAfterData","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:43.951Z | debug | Workflow execution finished successfully {"workflowId":"fksCUlSghdpu9QKb","file":"logger-proxy.js","function":"exports.debug"}
n8n-1  | 2025-11-03T07:37:43.953Z | debug | Executing hook (hookFunctionsSave) {"executionId":"91","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:37:43.954Z | debug | Save execution data to database for execution ID 91 {"executionId":"91","workflowId":"fksCUlSghdpu9QKb","finished":true,"stoppedAt":"2025-11-03T07:37:43.951Z","file":"shared-hook-functions.js","function":"updateExistingExecution"}
n8n-1  | 2025-11-03T07:37:43.993Z | debug | Executing hook (hookFunctionsPush) {"executionId":"91","pushRef":"rjpmbwqxpe","workflowId":"fksCUlSghdpu9QKb","file":"execution-lifecycle-hooks.js"}
n8n-1  | 2025-11-03T07:37:43.994Z | debug | Pushed to frontend: executionFinished {"dataType":"executionFinished","pushRefs":"rjpmbwqxpe","file":"abstract.push.js","function":"sendTo"}
n8n-1  | 2025-11-03T07:37:43.995Z | debug | Execution finalized {"executionId":"91","file":"active-executions.js","function":"finalizeExecution"}
n8n-1  | 2025-11-03T07:37:43.995Z | debug | Execution removed {"executionId":"91","file":"active-executions.js"}
n8n-1  | 2025-11-03T07:38:20.739Z | debug | Querying database for waiting executions {"scopes":["waiting-executions"],"file":"wait-tracker.js","function":"getWaitingExecutions"}
n8n-1  | 2025-11-03T07:39:20.739Z | debug | Querying database for waiting executions {"scopes":["waiting-executions"],"file":"wait-tracker.js","function":"getWaitingExecutions"}
n8n-1  | 2025-11-03T07:40:20.739Z | debug | Querying database for waiting executions {"scopes":["waiting-executions"],"file":"wait-tracker.js","function":"getWaitingExecutions"}
n8n-1  | 2025-11-03T07:41:20.739Z | debug | Querying database for waiting executions {"scopes":["waiting-executions"],"file":"wait-tracker.js","function":"getWaitingExecutions"}
n8n-1  | 2025-11-03T07:42:20.739Z | debug | Querying database for waiting executions {"scopes":["waiting-executions"],"file":"wait-tracker.js","function":"getWaitingExecutions"}
n8n-1  | 2025-11-03T07:43:20.739Z | debug | Querying database for waiting executions {"scopes":["waiting-executions"],"file":"wait-tracker.js","function":"getWaitingExecutions"}

danny22-seo commented:
@RomanDavydchuk
Hi, Is there any update?

RomanDavydchuk reviewed (approved):
LGTM

RomanDavydchuk committed:
Merge branch 'master' into add-proxy-agent-azure-embeddings
